packages/react-debug-tools/src/__tests__/ReactDevToolsHooksIntegration-test.js

```diff
@@ -46,6 +46,7 @@ describe('React hooks DevTools integration', () => {
     act = ReactTestRenderer.act;
   });
 
+  // @gate __DEV__
```

Comment:
Why this gating condition is required?

Reply:
This was due to the act usage failing in production env. There was already some gating here for the concurrent mode test case so I followed that pattern. But took a look again and updating the act import to the internal utility resolves all the gating on this module. Updated PR